controller/src/main/java/org/apache/rocketmq/controller/impl/manager/ReplicasInfoManager.java

```diff
@@ -201,6 +205,7 @@ public ControllerResult<ElectMasterResponseHeader> electMaster(
      * @param filter return true if the candidate is available
      * @return true if elect success
      */
+    @Deprecated
```

Comment:
If this method is no longer used, it can be deleted. It has not been released yet.

Reply:
done~